hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/mgmt/ui/MgmtUiAutoConfiguration.java

```diff
@@ -41,13 +41,18 @@
 @Import(MgmtUiConfiguration.class)
 public class MgmtUiAutoConfiguration {
 
+    @Bean
+    RedirectController uiRedirectController() {
```

Comment:
I would annotate it with `@ConditionalOnMissingBean` so it's possible to overwrite the redirect-controller in case you don't want to redirect `/` to `/UI/` like the default.